OPT_mission.Tanoa/sectorcontrol/functions/fnc_postinit.sqf

```diff
@@ -37,7 +37,7 @@ if (hasInterface) then {
         ["Flagge ziehen", "Flagge ziehen, wenn anvisiert."], 
         {
             if (
-                [cursorObject] in (GVARMAIN(nato_flags) + GVARMAIN(csat_flags)) and
+                cursorObject in (GVARMAIN(nato_flags) + GVARMAIN(csat_flags)) and
```

Comment:
Hier würde ich vllt lieber 
```
(cursorObject in GVARMAIN(nato_flags) or cursorObject in  GVARMAIN(csat_flags)) and ...
```
verwenden, da du mit dem `+`-Operator extra ein neues Array erstellst, nur um was darin zu suchen. Das ist okay für `foreach`-Schleifen, da die Alternative sonst ne ordentliche Code-Duplikation wäre, aber in diesem Fall kann man das mit überschaubarer Duplikation vermeiden.
Noch schneller wäre vllt sogar sich das `cursorObject` zuvor in eine Variable zu speichern, da der Aufruf von Variablen in der Regel deutlich schneller ist, als die Ausführung eines Befehls. Aber ich glaube das geht dann doch ein bisschen arg in Mikrooptimierungen über..